js/app.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+function reqResource() {
+  const genReq = new XMLHttpRequest();
+  const genReqListener = function() {
+    const genObj = JSON.parse(this.responseText);
+    const contContainer = document.querySelector('#contentContainer');
+
+    while (contContainer.firstChild) {
+      contContainer.removeChild(contContainer.firstChild);
+    }
+
+    if (this.status !== 200) {
```

Comment:
Try to implement this as a guard. Meaning that if you just return at the end of the if conditional, then the rest of the code won't execute. That means you can leave this section of code as its own section above the other else ifs.